intake/catalog/base.py

```diff
@@ -187,23 +183,31 @@ def __repr__(self):
 
     def __getattr__(self, item):
         if not item.startswith('_'):
-            return self._get_entry(item)
+            # Fall back to __getitem__.
+            try:
+                return self._get_entry(item)  # triggers reload_on_change
```

Comment:
So it is actually the same as `self[item]`, given that an attribute can't contain "."?

Reply:
Good point, will simplify.